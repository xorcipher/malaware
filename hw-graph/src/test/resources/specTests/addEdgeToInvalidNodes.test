# Tests that an edge can be added to two nodes within a graph when one or more of the specified nodes do not exist in the graph.

# Creates a new edge between two nodes.

CreateGraph graph

AddNode graph A

AddEdge graph A B edgeA