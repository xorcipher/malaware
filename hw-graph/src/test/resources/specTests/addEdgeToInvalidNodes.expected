# Tests that an edge can be added to two nodes within a graph when one or more of the specified nodes do not exist in the graph.

# Creates a new edge between two nodes.

created graph graph

added node A to graph

Exception: java.lang.IllegalArgumentException: Graph does not contain one of the specified parent/child nodes.